cilium-health/launch/launcher.go

```diff
@@ -35,7 +35,7 @@ var log = logging.DefaultLogger.WithField(logfields.LogSubsys, "cilium-health-la
 
 const (
 	serverProbeInterval  = 60 * time.Second
-	serverProbeDeadline  = 1 * time.Second
+	serverProbeDeadline  = 10 * time.Second
```

Comment:
This change seems unrelated to the rest of the commit. I'd recommend moving it to another commit maybe, and providing the motivation in the commit description.